quota/mysqlqm/mysql_quota.go

```diff
@@ -124,3 +130,29 @@ func countFromTable(ctx context.Context, db *sql.DB) (int, error) {
 	}
 	return count, nil
 }
+
+// turnOffInformationSchemaCache turn off statistics caching for MySQL 8
+// To always retrieve the latest statistics directly from the storage engine and bypass cached values, set information_schema_stats_expiry to 0.
+// See https://dev.mysql.com/doc/refman/8.0/en/server-system-variables.html#sysvar_information_schema_stats_expiry
+func turnOffInformationSchemaCache(ctx context.Context, db *sql.DB) error {
+	opt := "information_schema_stats_expiry"
+	res := db.QueryRowContext(ctx, "SHOW VARIABLES LIKE '%"+opt+"%'")
+	var none, expiry string
+	err := res.Scan(&none, &expiry)
```

Comment:
nit: Can you inline this with the check on `err` like this: `if err := res.Scan...; err != nil { ...`?

Reply:
ok